### ISSUE TITLE
Has "output" at the beginning of each filename

### ISSUE THREAD

**Issue by swindlesmccoop** [Tupelov/Root_E-Renamer#1]
probably has to do with line 34:
`File rename = new File("./output"+list.get(i)+".wav");`
maybe change to
`File rename = new File("./"+list.get(i)+".wav");`?

**Comment by Tupelov** [comment]
Yeah I knew about this bug but I must've forgot to fix it despite already knowing what I had to fix, I can make the commit but it might take me 1 1/2 weeks to push a new release so youll have to build it yourself from source.

**Comment by swindlesmccoop** [comment]
mfw when still no build
